service/dynamodb/dynamodbattribute/decode.go

```diff
@@ -454,6 +455,18 @@ func (d *Decoder) decodeString(s *string, v reflect.Value, fieldTag tag) error {
 		return d.decodeNumber(s, v)
 	}
 
+	// To maintain backwards compatibility with ConvertFrom family of methods which
+	// converted strings to time.Time structs (using JSON encoding/decoding)
+	if t, ok := v.Interface().(time.Time); ok {
+		b := []byte(*s)
+		err := t.UnmarshalJSON(b)
```

Comment:
Thanks @spaceweasel. I believe Parse and Format will make for more readable code.